packages/docs/components/layout/mdx-provider-components.js

```diff
@@ -1,13 +1,15 @@
 import { Title, Card, CardBody } from '@patternfly/react-core';
 import { createUseStyles } from 'react-jss';
 import classnames from 'classnames';
+import Link from 'next/link';
 
 const useTableStyles = createUseStyles({
     card: {
         overflowY: 'auto'
     }
 });
 
+const A = ({ children, target,  ...props }) => <Link {...props}><a target={target}>{children}</a></Link>;
```

Comment:
Shouldn't you also provide a `href` to the `a` element? 🤔

Reply:
No, the href is injected by the link. However, it seems that it consumes the target property, maybe others aswell